File: Module-CP/Assignments/assignment14/Pass_by_value/q2.c
#include<stdio.h>
typedef struct Employee{
    int id;
    char name[20];
    int salary;
}Employee;
Employee storeEmployee();
void display(Employee);
int main(){
    Employee e1,e2;
    e1=storeEmployee();
    e2=storeEmployee();

    printf("Details For Employee-1");
    display(e1);
    printf("\nDetails For Employee-2");
    display(e2);
}
Employee storeEmployee(){
      Employee e;
      printf("Enter ID:");
      scanf("%d",&e.id);
      printf("Enter Name:");
      scanf("%s",e.name);
      printf("Enter Salary:");
      scanf("%d",&e.salary);

      return e;
}
void display(Employee e){
    printf("\nEmployee Id:%d",e.id);
    printf("\nEmployee Name:%s",e.name);
    printf("\nEmployee Salary:%d",e.salary);
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/By_using_Functions/q2.java

import java.util.*;

class Employee {

    int id;
    String name;
    float salary;

    Scanner sc = new Scanner(System.in);

    void setId(int id) {
        this.id = id;
    }

    void setName(String name) {
        this.name = name;
    }

    void setSalary(float salary) {
        this.salary = salary;
    }

    int getId() {
        return this.id;
    }

    String getName() {
        return this.name;
    }

    float getSalary() {
        return this.salary;
    }

    void display() {
        System.out.println("Employee Name:" + this.name);
        System.out.println("Employee ID:" + this.id);
        System.out.println("Employee Salary:" + this.salary);
    }
}

public class q2 {

    public static void main(String[] args) {
        Employee e1 = new Employee();
        Employee e2 = new Employee();

        System.out.println("Fill the Details of Employee:");
        e1.setName("Pratik");
        e1.setId(101);
        e1.setSalary(50000);

        e2.setName("Rahul");
        e2.setId(102);
        e2.setSalary(60000);

        System.out.println("Details of Employee:");
        e1.display();
        e2.display();
    }
}
